src/vivarium_public_health/metrics/utilities.py

```diff
@@ -80,18 +80,18 @@ def get_output_template(by_age: bool, by_sex: bool, by_year: bool) -> Template:
         A template string with measure and possibly additional criteria.
 
     """
-    template = '{measure}'
+    template = '${measure}'
```

Comment:
I haven't really used template strings before so why do you need the $?

Reply:
Holdover from old string formatting, I think.  I was surprised too.  It's just the escape character used here.  I think we could customize that, but not worth it.